examples/src/components/MultipleDomainsExample.jsx

```diff
@@ -0,0 +1,91 @@
+        this.setSdk2 = this.setSdk2.bind(this);
+    }
+
+    setSdk2(e) {
+        const formRef = e.target;
+        const inputs = formRef.querySelectorAll('[name]');
+        const inputsByName = keyBy(inputs, 'name');
+        const formValues = mapValues(inputsByName, 'value');
+
+        const sdk2 = factory({ domain: formValues.domain });
+        sdk2.user.login(formValues.sdk2Username, formValues.sdk2Password)
+            .then(() => {
+                this.setState({ sdk2LoginError: '' });
+                this.setState({ sdk2, sdk2Config: formValues });
+            }).catch((error) => {
+                this.setState({ sdk2LoginError: error.toString() });
+                console.log(error); // eslint-disable-line no-console
+            });
+
+        e.preventDefault();
+        return false;
+    }
+
+    render() {
+        return (
+            <div>
+                <h2>Default SDK:</h2>
+                <Kpi
+                    projectId={projectId}
+                    measure={totalSalesIdentifier}
+                />
+
+                <hr className="separator" />
+                <h2>Second SDK:</h2>
+                <Kpi
+                    projectId={this.state.sdk2Config.projectId}
+                    measure={this.state.sdk2Config.measureId}
+                    sdk={this.state.sdk2}
+                />
+
+                <div className="alert">{this.state.sdk2LoginError}</div>
+
+                <form action="?" method="get" onSubmit={this.setSdk2}>
+                    <p><strong>Login to second SDK first:</strong></p>
+                    Careful - cookies ignore port, you have to use different hostname to log in.<br />
+
+                    <table>
+                        <tbody>
+                            <tr><td>Domain: </td><td><input type="text" name="domain" /></td></tr>
+                            <tr><td>Username: </td><td><input type="text" name="sdk2Username" /></td></tr>
+                            <tr><td>Password: </td><td><input type="password" name="sdk2Password" /></td></tr>
+                            <tr><td>KPI project id: </td><td><input type="text" name="projectId" /></td></tr>
+                            <tr><td>KPI measure id: </td><td><input type="text" name="measureId" /></td></tr>
+                        </tbody>
+                    </table>
+                    <input type="submit" className="btn" />
+
+                </form>
+                <pre>{`
+                // WebApp can't send Access-Control-Allow-Headers, so it needs a proxy with CORS (options+get+post) like this:
+                res.header('Access-Control-Allow-Origin', 'https://localhost:8999');
+                res.header('Access-Control-Allow-Headers', 'content-type, x-gdc-js-pkg, x-gdc-js-pkg-version, x-gdc-js-sdk-comp, x-gdc-js-sdk-comp-props');
+                res.header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS');
+                res.header('Access-Control-Allow-Credentials', true);
```

Comment:
Can we add this proxy to examples and therefore enable this example to be integration tested?

Reply:
If it is too much effort, I am OK with making a separate ticket for it.